src/signature/src/main.rs

```diff
@@ -0,0 +1,28 @@
+use ethers::prelude::*;
+
+#[tokio::main]
+async fn main() {
+    let wallet = LocalWallet::new(&mut rand::thread_rng());
+    println!("private key: {:?}", wallet.signer().to_bytes().to_vec());
+
+    // ======== generate evm registry tx ========
+    let tx = TransactionRequest::new()
+        .from(wallet.address())
+        .to("0xb0e5863d0ddf7e105e409fee0ecc0123a362e14b") // MINTER_ADDRESS
```

Comment:
this information should be retrieved by calling `registration_ic_agent_info`

Reply:
fixed